communities/src/tests.rs

```diff
@@ -378,8 +313,9 @@ fn remove_location_works() {
         // remove second location
         EncointerCommunities::remove_location(Origin::signed(some_bootstrapper.clone()), cid, location2).ok();
         assert_eq!(EncointerCommunities::locations(&cid, &geo_hash), vec![]);
-        assert_eq!(EncointerCommunities::cids_by_geohash(&geo_hash), vec![]);
-
+        let mut vector = vec![CommunityIdentifier::default()];
```

Comment:
Is this because the compiler can't infer the type?

if yes, use:
```
let vec: Vec<CommunityIdentifier> = vec![];
```
or
```
let vec = Vec::<CommunityIdentifier>::new();
```